.github/workflows/go-mod-cache.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
       - name: Install Dependencies
         shell: bash
-        run:  go mod download all
+        run:  go mod download
```

Comment:
Including `all` is apparently some superset of packages that we don't actually need, which includes `github.com/sourcegraph/sourcegraph/lib`